src/Json/JsonValue.fs

```diff
@@ -155,6 +155,23 @@ type private JsonParser(jsonText:string) =
     let skipWhitespace() =
       while i < s.Length && Char.IsWhiteSpace s.[i] do
         i <- i + 1
+    let skipLine() =
+        while i < s.Length && not (s.[i] = '\n') do
+            i <- i+1
+        i <- i+1
+        skipWhitespace()
+
+    let skipMultiLineComment()=
+        while i < s.Length && not (String.Equals(s.[i..i+1],"*/")) do
```

Comment:
Will this not go boom 💥 if `i` gets to `s.Length-1`?

Reply:
Hi,
Case-1: white spaces after comments are handled by skipwhitespaces(), that is called after a new line character is found
Case-2: for a single line comment, the terminating condition is new line character, so it skips until it finds that. 
If i=s.length-1 and is still looking for coment terminating chars, then it's anyway wrong JSON format,

Thanks for the comments, I will add test cases for all these suggestions